bin/run_nwp_preparation.py

```diff
@@ -41,9 +41,8 @@
 
 def prepare_and_publish(pub, options, flens):
     config_file_name = options.config_file
-    starttime = datetime.utcnow() - timedelta(days=1)
+    starttime = datetime.utcnow() - timedelta(days=8)
```

Comment:
Should it really check for "new" files for the last 8 days?

Reply:
Oups, checked in this by mistake :)!